src/mito2/src/request.rs

```diff
@@ -156,10 +191,12 @@ impl WriteRequest {
                     InvalidRequestSnafu {
                         region_id,
                         reason: format!(
-                            "column {} expect type {:?}, given: {:?}({})",
+                            "column {} expect type {:?}, given: {}({})",
```

Comment:
Looks like this check method doesn't handle duplicate column names. Will it happen?

Reply:
`WriteRequest::new` already ensures this.